pkg/pillar/hypervisor/kvm.go

```diff
@@ -1001,65 +1382,32 @@ func multifunctionDevGroup(pcis []pciDevice) multifunctionDevs {
 	return mds
 }
 
-func (p pciDevice) pciLongWOFunction() (string, error) {
-	pciLongSplit := strings.Split(p.pciLong, ".")
-	if len(pciLongSplit) == 0 {
-		return "", fmt.Errorf("could not split %s", p.pciLong)
-	}
-	pciWithoutFunction := strings.Join(pciLongSplit[0:len(pciLongSplit)-1], ".")
-
-	return pciWithoutFunction, nil
-}
-
 type pciAssignmentsTemplateFiller struct {
 	multifunctionDevices multifunctionDevs
 	file                 io.Writer
 }
 
 func (f *pciAssignmentsTemplateFiller) pciEBridge(pciID int, pciWOFunction string) error {
-	pciChildTemplateVars := struct {
-		PCIId int
-		Bus   int
-	}{
-		PCIId: 0,
-		Bus:   pciID,
-	}
-
-	tPCIeBridge, err := template.New("qemuPCIeBridge").Parse(qemuPCIPassthruBridgeTemplate)
-	if err != nil {
-		return fmt.Errorf("parsing qemuPCIPassthruBridgeTemplate failed: %w", err)
-	}
-	if err := tPCIeBridge.Execute(f.file, pciChildTemplateVars); err != nil {
+	pcieBridgeContext := tQemuPCIeBridgeContext{Bus: pciID}
```

Comment:
I'm a bit confused about how we use pciID as the Bus address and do not fill PICId in the pcieBridgeContext at all (it's always 0).

Reply:
Yeah, it used to be the same, I see... But it's still confusing))

Reply:
But yeah, it makes total sense...

Reply:
Do you want me to try to make it less confusing or has the confusion level reached acceptable levels? :)

Reply:
Nah... It's not critical. Actually an example of expected config sections (with note which fields are expected to correspond to wich) in the doc or here would help, but it's up to you.

Reply:
I have simplified the bridge template to avoid the confusion.